packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -3,11 +3,12 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Fixed Drawer menu not closing in Internet Explorer on clicking outside of drawer menu.
```

Comment:
is this part of your change ?

Reply:
This change, even-though not a part of this PR, is an intended change, since this changelog entry was incorrectly merged as released.